packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -261,11 +261,16 @@ export class ValidatorApi implements IValidatorApi {
     if (!this.sync.isSynced()) {
       let syncStatus;
       try {
-        syncStatus = this.config.types.SyncingStatus.toJson(await this.sync.getSyncStatus());
+        syncStatus = await this.sync.getSyncStatus();
```

Comment:
Just an opinionated comment, is
```ts
const syncStatus = await this.sync.getSyncStatus().catch(e => {
  throw new ApiError(503, "Node is stopped");
})
```
cleaner?

Reply:
yeah, this logic is still likely wrong. Lets fix it in another PR.

Reply:
alternatively
```typescript
try {
  const syncStatus . await this.sync.getSyncStatus();
  ...
} catch (e) {
  throw new ApiError(503, "Node is stopped");
}
```

Reply:
Also, you should log the error so we are not helpless with the non-specific "Node is stopped" error

Reply:
https://github.com/ChainSafe/lodestar/issues/1492